lib/pages/homeserver_picker/homeserver_picker.dart

```diff
@@ -34,7 +34,7 @@ class HomeserverPicker extends StatefulWidget {
 
 class HomeserverPickerController extends State<HomeserverPicker> with ConnectPageMixin {
   
-  HomeserverState state = HomeserverState.enterServerName;
+  HomeserverState state = HomeserverState.ssoLoginServer;
```

Comment:
please note that, we also support 3rd party server. Agree, we can bypass the unnecessary click for user when `sso` support, but only bypass when we know this server accepts SSO

Reply:
Updated, pls check demo